### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by SumiranAgg**
- Adding app url after deployment
- Doc link in release notes
- helptext for FILE_IN_USE

### CODE REVIEW

File: Tasks/AzureRmWebAppDeploymentV4/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -221,5 +221,6 @@
   "loc.messages.ASE_WebDeploySSLIssueRecommendation": "To use a certificate in App Service, the certificate must be signed by a trusted certificate authority. If your web app gives you certificate validation errors, you're probably using a self-signed certificate and to resolve them you need to pass -allowUntrusted in additional arguments of web deploy option.",
   "loc.messages.FailedToGetResourceID": "Failed to get resource ID for resource type '%s' and resource name '%s'. Error: %s",
   "loc.messages.JarPathNotPresent": "Java jar path is not present",
-  "loc.messages.FailedToUpdateApplicationInsightsResource": "Failed to update Application Insights '%s' Resource. Error: %s"
+  "loc.messages.FailedToUpdateApplicationInsightsResource": "Failed to update Application Insights '%s' Resource. Error: %s",
+  "loc.messages.RunFromZipPreventsFileInUseError": "Move from Web Deploy to RunFrom Package, which helps in avoiding FILE_IN_USE error. Note that Run From Package does not support msBuild package type. Please change your package format to use this deployment method."
```
Comment (vincentdass) [inline Tasks/AzureRmWebAppDeploymentV4/Strings/resources.resjson/en-US/resources.resjson]: Can you please get this reviewed from PM too?

File: Tasks/AzureRmWebAppDeploymentV4/deploymentProvider/AzureRmWebAppDeploymentProvider.ts [hunk 2]
```diff
@@ -54,6 +50,10 @@ export class AzureRmWebAppDeploymentProvider implements IWebAppDeploymentProvide
             this.activeDeploymentID = await this.kuduServiceUtility.updateDeploymentStatus(isDeploymentSuccess, null, {'type': 'Deployment', slotName: this.appService.getSlot()});
             tl.debug('Active DeploymentId :'+ this.activeDeploymentID);
         }
+        let appServiceApplicationUrl: string = await this.appServiceUtility.getApplicationURL(!this.taskParams.isLinuxApp
```
Comment (vincentdass) [inline Tasks/AzureRmWebAppDeploymentV4/deploymentProvider/AzureRmWebAppDeploymentProvider.ts]: nit: new line after }

File: Tasks/AzureRmWebAppDeploymentV4/task.json [hunk 3]
```diff
@@ -18,9 +18,9 @@
     "version": {
         "Major": 4,
         "Minor": 3,
-        "Patch": 13
+        "Patch": 14
     },
-    "releaseNotes": "What's new in version 4.* (preview)<br />Supports Zip Deploy, Run From Package, War Deploy<br />Supports App Service Environments<br />Improved UI for discovering different App service types supported by the task<br/>Run From Package is the preferred deployment method, which makes files in wwwroot folder read-only<br/>Click [here](https://aka.ms/azurermwebdeployreadme) for more information.",
+    "releaseNotes": "What's new in version 4.* (preview)<br />Supports Zip Deploy, Run From Package, War Deploy [Details here](https://docs.microsoft.com/en-us/azure/app-service/app-service-deploy-zip)<br />Supports App Service Environments<br />Improved UI for discovering different App service types supported by the task<br/>Run From Package is the preferred deployment method, which makes files in wwwroot folder read-only<br/>Click [here](https://aka.ms/azurermwebdeployreadme) for more information.",
```
Comment (vincentdass) [inline Tasks/AzureRmWebAppDeploymentV4/task.json]: Can we get a fwd link `aka.ms`?

File: Tasks/AzureRmWebAppDeploymentV4/deploymentProvider/AzureRmWebAppDeploymentProvider.ts (hunk 2 above)
Comment (SumiranAgg) [reply Tasks/AzureRmWebAppDeploymentV4/deploymentProvider/AzureRmWebAppDeploymentProvider.ts]: Done

**Pull request merged**